tests/test_install.ts

```diff
@@ -96,7 +96,7 @@ describe("Agama test", function () {
         });
         page = await browser.newPage();
         page.setDefaultTimeout(20000);
-        await page.goto(agamaServer, { timeout: 60000, waitUntil: "domcontentloaded" });
+        await page.goto(agamaServer, { timeout: 60000, waitUntil: "networkidle2" });
```

Comment:
this is unclear to me if it is needed.

Reply:
Not really needed I will switch again on a next PR